.eslintrc.js

```diff
@@ -19,6 +19,7 @@
     "jest",
     "react-hooks"
   ],
+  "reportUnusedDisableDirectives": true,
```

Comment:
this produces warnings when executed without `--report-unused-disable-directives`